drivers/SmartThings/zwave-thermostat/src/init.lua

```diff
@@ -87,6 +102,32 @@ local function set_setpoint_factory(setpoint_type)
   end
 end
 
+local function setpoint_capabilites_report(driver, device, cmd)
+  local function concrete_setpoint_temperature(precision, value)
+    local result = value / (10 ^ precision)
+    return result
+  end
```

Comment:
we have a little bit of magic that we do in our generated z-wave libraries so that fields encoded with precision and size are just automatically translated to numbers, so you should just be able to use `cmd.args.min_value` and `cmd.args.max_value` without having to do the manual precision calculation.